filebeat/input/journald/input.go

```diff
@@ -303,3 +352,64 @@ func (r *readerAdapter) Next() (reader.Message, error) {
 
 	return m, nil
 }
+
+// parseSystemdVersion parses the string version from Systemd fetched via D-Bus.
+// The function will parse and return the 3 digit major version, minor version
+// and patch are ignored.
+func parseSystemdVersion(ver string) (int, error) {
+	re := regexp.MustCompile(`(v)?(?P<version>\d\d\d)(\.)?`)
```

Comment:
the 3 digits makes me a little nervous, if systemd goes to 4 this all breaks.

`^(?P<version>\d{3,})` works with the examples provided.  thoughts?